packages/storage-plugin/src/symbols.ts

```diff
@@ -7,6 +7,23 @@ export const enum StorageOption {
   SessionStorage
 }
 
+export interface NgxsStoragePluginSerializationOptions {
```

Comment:
@davidmorissette I'm just wondering if it would be possible to add any additional typings here?
Not sure how this would work when expressed in the array, but let's hope that it 'Just Works™️ '.
We could default it to `any`... maybe that would make the array happy.
Something like this (comments excluded for berevity):
```ts
export interface NgxsStoragePluginSerializationOptions<TStateModel = any, TStoredModel = any> {
  key?: string;
  onBeforeSerialize?: (obj: TStateModel) => TStoredModel;
  onAfterDeserialize?: (obj: TStoredModel) => TStateModel;
}
```

Reply:
@davidmorissette ping

Reply:
@markwhitfeld As far as I know, it is not possible to implement an array of different generics with TypeScript. Please correct me if I am wrong. Plus, I don't see any pattern of reusability with these methods and I am not so sure generics are a good fit there. Migrations don't make usage of them.